### ISSUE TITLE
Could not find a version that satisfies the requirement pywin32>=223 (from pypiwin32>=223->web3)

### ISSUE THREAD

**Issue by artyomLisovskij** [ethereum/web3.py#1052]
* Version: 4.7.0
* Python: 3.5
* OS: linux

### What was wrong?

Can't install via pip
```
user@b41606c8a974:/projects$ sudo pip3 install --upgrade pip

Collecting pip

Downloading https://files.pythonhosted.org/packages/5f/25/e52d3f31441505a5f3af41213346e5b6c221c9e086a166f3703d2ddaf940/pip-18.0-py2.py3-none-any.whl (1.3MB)
    100% |████████████████████████████████| 1.3MB 22.9MB/s

Installing collected packages: pip

Found existing installation: pip 10.0.1
    Uninstalling pip-10.0.1:
      Successfully uninstalled pip-10.0.1

Successfully installed pip-18.0

user@b41606c8a974:/projects$ pip3 install --upgrade pip

Requirement already up-to-date: pip in /usr/local/lib/python3.5/site-packages (18.0)
user@b41606c8a974:/projects$ pip3 install web3

Collecting web3

  Using cached https://files.pythonhosted.org/packages/e5/f9/d0150ac1a90cdc6312426b3be8ffb948e1ca88f8b79898be1f4140b02f45/web3-4.7.0-py3-none-any.whl

Collecting eth-hash[pycryptodome]<1.0.0,>=0.2.0 (from web3)

  Using cached https://files.pythonhosted.org/packages/0d/d8/0f0c8d4ccaa4c8d25524fd1ed9f5c6d9551b52b74b973f338de3f0a23111/eth_hash-0.2.0-py3-none-any.whl

Collecting eth-abi<2.0.0,>=1.2.0 (from web3)

  Using cached https://files.pythonhosted.org/packages/3d/4e/66d711f39c892c295cd61d2a241d26e4fa253e483382bdb34e39ea4c4d96/eth_abi-1.2.0-py3-none-any.whl

Collecting eth-utils<2.0.0,>=1.0.1 (from web3)
  Using cached https://files.pythonhosted.org/packages/a8/ac/d7df3ce575e8251b7d6736fedc336b5475df28a77e8aa1ad660954f91420/eth_utils-1.2.1-py3-none-any.whl

Collecting cytoolz<1.0.0,>=0.9.0 (from web3)
  Using cached https://files.pythonhosted.org/packages/36/f4/9728ba01ccb2f55df9a5af029b48ba0aaca1081bbd7823ea2ee223ba7a42/cytoolz-0.9.0.1.tar.gz

Collecting toolz<1.0.0,>=0.9.0 (from web3)
  Using cached https://files.pythonhosted.org/packages/14/d0/a73c15bbeda3d2e7b381a36afb0d9cd770a9f4adc5d1532691013ba881db/toolz-0.9.0.tar.gz

Collecting pypiwin32>=223 (from web3)
  Using cached https://files.pythonhosted.org/packages/d0/1b/2f292bbd742e369a100c91faa0483172cd91a1a422a6692055ac920946c5/pypiwin32-223-py3-none-any.whl

Collecting eth-account<0.4.0,>=0.2.1 (from web3)
  Using cached https://files.pythonhosted.org/packages/a3/d8/6118bc0e00d6ec092bb02b012989d1018893755eb84da53266412ac359e3/eth_account-0.3.0-py3-none-any.whl

Collecting websockets<7.0.0,>=6.0.0 (from web3)
  Using cached https://files.pythonhosted.org/packages/8e/e8/8c959153f7e36258b410b90869c46a3f881a5a345539bf6b2a16049ea5f0/websockets-6.0-cp35-cp35m-manylinux1_x86_64.whl

Collecting lru-dict<2.0.0,>=1.1.6 (from web3)
  Using cached https://files.pythonhosted.org/packages/00/a5/32ed6e10246cd341ca8cc205acea5d208e4053f48a4dced2b1b31d45ba3f/lru-dict-1.1.6.tar.gz

Collecting hexbytes<1.0.0,>=0.1.0 (from web3)
  Using cached https://files.pythonhosted.org/packages/18/bd/21697d93ee2153c8c12253262b740f571e94341d4cee411504032fff3f56/hexbytes-0.1.0-py3-none-any.whl

Collecting requests<3.0.0,>=2.16.0 (from web3)
  Using cached https://files.pythonhosted.org/packages/65/47/7e02164a2a3db50ed6d8a6ab1d6d60b69c4c3fdf57a284257925dfc12bda/requests-2.19.1-py2.py3-none-any.whl

Collecting pycryptodome<4,>=3.6.6; extra == "pycryptodome" (from eth-hash[pycryptodome]<1.0.0,>=0.2.0->web3)
  Using cached https://files.pythonhosted.org/packages/e9/02/f22c807b475c872117e53d087b7c122980269bf32f54880b6521fbf378ad/pycryptodome-3.6.6-cp35-cp35m-manylinux1_x86_64.whl

Collecting parsimonious==0.8.0 (from eth-abi<2.0.0,>=1.2.0->web3)
  Using cached https://files.pythonhosted.org/packages/4a/89/32c55944cd30dff856f16859ee325b13c83c260d0c56c0eed511e8063c87/parsimonious-0.8.0.tar.gz

Collecting eth-typing<=2 (from eth-abi<2.0.0,>=1.2.0->web3)
  Using cached https://files.pythonhosted.org/packages/5f/cb/e941031a77d19eef05f8bcdd40f6d8a3b55b7fa45aa7100206d64216c29b/eth_typing-1.3.0-py3-none-any.whl

Collecting pywin32>=223 (from pypiwin32>=223->web3)
  Could not find a version that satisfies the requirement pywin32>=223 (from pypiwin32>=223->web3) (from versions: )
No matching distribution found for pywin32>=223 (from pypiwin32>=223->web3)```

**Comment by artyomLisovskij** [comment]
without pip caches:
```
$ pip3 install --no-cache-dir web3
Collecting web3
  Downloading https://files.pythonhosted.org/packages/e5/f9/d0150ac1a90cdc6312426b3be8ffb948e1ca88f8b79898be1f4140b02f45/web3-4.7.0-py3-none-any.whl (122kB)
    100% |████████████████████████████████| 122kB 13.1MB/s
Collecting eth-account<0.4.0,>=0.2.1 (from web3)
  Downloading https://files.pythonhosted.org/packages/a3/d8/6118bc0e00d6ec092bb02b012989d1018893755eb84da53266412ac359e3/eth_account-0.3.0-py3-none-any.whl
Collecting eth-abi<2.0.0,>=1.2.0 (from web3)
  Downloading https://files.pythonhosted.org/packages/3d/4e/66d711f39c892c295cd61d2a241d26e4fa253e483382bdb34e39ea4c4d96/eth_abi-1.2.0-py3-none-any.whl
Collecting requests<3.0.0,>=2.16.0 (from web3)
  Downloading https://files.pythonhosted.org/packages/65/47/7e02164a2a3db50ed6d8a6ab1d6d60b69c4c3fdf57a284257925dfc12bda/requests-2.19.1-py2.py3-none-any.whl (91kB)
    100% |████████████████████████████████| 92kB 25.2MB/s
Collecting hexbytes<1.0.0,>=0.1.0 (from web3)
  Downloading https://files.pythonhosted.org/packages/18/bd/21697d93ee2153c8c12253262b740f571e94341d4cee411504032fff3f56/hexbytes-0.1.0-py3-none-any.whl
Collecting websockets<7.0.0,>=6.0.0 (from web3)
  Downloading https://files.pythonhosted.org/packages/8e/e8/8c959153f7e36258b410b90869c46a3f881a5a345539bf6b2a16049ea5f0/websockets-6.0-cp35-cp35m-manylinux1_x86_64.whl (86kB)
    100% |████████████████████████████████| 92kB 35.6MB/s
Collecting pypiwin32>=223 (from web3)
  Downloading https://files.pythonhosted.org/packages/d0/1b/2f292bbd742e369a100c91faa0483172cd91a1a422a6692055ac920946c5/pypiwin32-223-py3-none-any.whl
Collecting eth-hash[pycryptodome]<1.0.0,>=0.2.0 (from web3)
  Downloading https://files.pythonhosted.org/packages/0d/d8/0f0c8d4ccaa4c8d25524fd1ed9f5c6d9551b52b74b973f338de3f0a23111/eth_hash-0.2.0-py3-none-any.whl
Collecting lru-dict<2.0.0,>=1.1.6 (from web3)
  Downloading https://files.pythonhosted.org/packages/00/a5/32ed6e10246cd341ca8cc205acea5d208e4053f48a4dced2b1b31d45ba3f/lru-dict-1.1.6.tar.gz
Collecting toolz<1.0.0,>=0.9.0 (from web3)
  Downloading https://files.pythonhosted.org/packages/14/d0/a73c15bbeda3d2e7b381a36afb0d9cd770a9f4adc5d1532691013ba881db/toolz-0.9.0.tar.gz (45kB)
    100% |████████████████████████████████| 51kB 29.1MB/s
Collecting eth-utils<2.0.0,>=1.0.1 (from web3)
  Downloading https://files.pythonhosted.org/packages/a8/ac/d7df3ce575e8251b7d6736fedc336b5475df28a77e8aa1ad660954f91420/eth_utils-1.2.1-py3-none-any.whl
Collecting cytoolz<1.0.0,>=0.9.0 (from web3)
  Downloading https://files.pythonhosted.org/packages/36/f4/9728ba01ccb2f55df9a5af029b48ba0aaca1081bbd7823ea2ee223ba7a42/cytoolz-0.9.0.1.tar.gz (443kB)
    100% |████████████████████████████████| 450kB 35.4MB/s
Collecting eth-keyfile<0.6.0,>=0.5.0 (from eth-account<0.4.0,>=0.2.1->web3)
  Downloading https://files.pythonhosted.org/packages/eb/a5/3615d100b62fbf20fe5d5c0d1d4d7326eac861d260b0cd2a36af2bf8ccb1/eth_keyfile-0.5.1-py3-none-any.whl
Collecting eth-keys<0.3.0,>=0.2.0b3 (from eth-account<0.4.0,>=0.2.1->web3)
  Downloading https://files.pythonhosted.org/packages/9b/96/bec507291a2ecac65358b4485282ce02e8c0cdd30980a232791b355a1f5b/eth_keys-0.2.0b3-py3-none-any.whl
Collecting attrdict<3,>=2.0.0 (from eth-account<0.4.0,>=0.2.1->web3)
  Downloading https://files.pythonhosted.org/packages/35/bb/bac3e42ae04bc082c28cd8186bfb5b50fb240a4f7419f876c683125ccc8b/attrdict-2.0.0.tar.gz
Collecting eth-rlp<1,>=0.1.2 (from eth-account<0.4.0,>=0.2.1->web3)
  Downloading https://files.pythonhosted.org/packages/56/4b/b309fb44dd9bb975e84f3d626392eec1eeed0108e55a51cd7dd8956a73ad/eth_rlp-0.1.2-py3-none-any.whl
Collecting parsimonious==0.8.0 (from eth-abi<2.0.0,>=1.2.0->web3)
  Downloading https://files.pythonhosted.org/packages/4a/89/32c55944cd30dff856f16859ee325b13c83c260d0c56c0eed511e8063c87/parsimonious-0.8.0.tar.gz
Collecting eth-typing<=2 (from eth-abi<2.0.0,>=1.2.0->web3)
  Downloading https://files.pythonhosted.org/packages/5f/cb/e941031a77d19eef05f8bcdd40f6d8a3b55b7fa45aa7100206d64216c29b/eth_typing-1.3.0-py3-none-any.whl
Collecting certifi>=2017.4.17 (from requests<3.0.0,>=2.16.0->web3)
  Downloading https://files.pythonhosted.org/packages/df/f7/04fee6ac349e915b82171f8e23cee63644d83663b34c539f7a09aed18f9e/certifi-2018.8.24-py2.py3-none-any.whl (147kB)
    100% |████████████████████████████████| 153kB 39.6MB/s
Collecting chardet<3.1.0,>=3.0.2 (from requests<3.0.0,>=2.16.0->web3)
  Downloading https://files.pythonhosted.org/packages/bc/a9/01ffebfb562e4274b6487b4bb1ddec7ca55ec7510b22e4c51f14098443b8/chardet-3.0.4-py2.py3-none-any.whl (133kB)
    100% |████████████████████████████████| 143kB 38.8MB/s
Collecting idna<2.8,>=2.5 (from requests<3.0.0,>=2.16.0->web3)
  Downloading https://files.pythonhosted.org/packages/4b/2a/0276479a4b3caeb8a8c1af2f8e4355746a97fab05a372e4a2c6a6b876165/idna-2.7-py2.py3-none-any.whl (58kB)
    100% |████████████████████████████████| 61kB 37.9MB/s
Collecting urllib3<1.24,>=1.21.1 (from requests<3.0.0,>=2.16.0->web3)
  Downloading https://files.pythonhosted.org/packages/bd/c9/6fdd990019071a4a32a5e7cb78a1d92c53851ef4f56f62a3486e6a7d8ffb/urllib3-1.23-py2.py3-none-any.whl (133kB)
    100% |████████████████████████████████| 143kB 41.5MB/s
Collecting pywin32>=223 (from pypiwin32>=223->web3)
  Could not find a version that satisfies the requirement pywin32>=223 (from pypiwin32>=223->web3) (from versions: )
No matching distribution found for pywin32>=223 (from pypiwin32>=223->web3)
```

**Comment by voith** [comment]
Yup been fighting it myself with this for sometime. As a quick fix install `4.6.0` till the team actually fixes this.

**Comment by RomanUsov** [comment]
I have the same problem as artyomLisovskij.

**Comment by RomanUsov** [comment]
I have managed this issue.
In my case:
* Install python-dev `sudo apt install python3.6-dev` (I use Ubuntu 16.04 and python 3.6)
* Clone repo `git clone git@github.com:ethereum/web3.py.git`
* cd to web3 folder `cd web3.py`
* `pip install .`
After these steps I have installed web 4.7 version

```
$ pip freeze
attrdict==2.0.0
certifi==2018.8.24
chardet==3.0.4
cytoolz==0.9.0.1
eth-abi==1.2.0
eth-account==0.3.0
eth-hash==0.2.0
eth-keyfile==0.5.1
eth-keys==0.2.0b3
eth-rlp==0.1.2
eth-typing==1.3.0
eth-utils==1.2.1
hexbytes==0.1.0
idna==2.7
lru-dict==1.1.6
parsimonious==0.8.0
pycryptodome==3.6.6
requests==2.19.1
rlp==1.0.2
six==1.11.0
toolz==0.9.0
urllib3==1.23
web3==4.7.0
websockets==6.0

**Comment by carver** [comment]
Sorry for the downtime in prod! This is now fixed in v4.7.1. See #1053 for more explanation.
